### PR TITLE
Add Axone PNOR Layout XML

### PR DESCRIPTION
**Pull request by mabaiocchi**
This new PNOR layout file should be used for all Axone-based configurations,
like the forthcoming Swift config.

### CODE REVIEW

**Comment by mabaiocchi** [comment]
@wghoffa and @dcrowell77 This is create a new PNOR layout for Axone.  It's basis is what was at our internal hostboot master src/build/buildpnor/pnorLayoutAxone.xml file as of yesterday.

File: p9Layouts/axonePnorLayout_64.xml [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+    <section>
+        <description>Hostboot Runtime Services for Sapphire (7.0MB)</description>
+        <eyeCatch>HBRT</eyeCatch>
+        <physicalOffset>0x162D000</physicalOffset>
+        <physicalRegionSize>0x700000</physicalRegionSize>
+        <sha512Version/>
+        <side>sideless</side>
+        <ecc/>
+    </section>
+    <section>
```
Comment (dcrowell77) [inline p9Layouts/axonePnorLayout_64.xml]: Switch out the PAYLOAD to match what is being used in the other layouts (BOOTKERNEL, etc)

File: p9Layouts/axonePnorLayout_64.xml [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+        <description>Payload (19.875MB)</description>
+        <eyeCatch>PAYLOAD</eyeCatch>
+        <physicalOffset>0x1D2D000</physicalOffset>
+        <physicalRegionSize>0x13E0000</physicalRegionSize>
+        <sha512Version/>
+        <side>sideless</side>
+        <ecc/>
+    </section>
+    <section>
```
Comment (dcrowell77) [inline p9Layouts/axonePnorLayout_64.xml]: No test sections in the public version

File: p9Layouts/axonePnorLayout_64.xml [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+        <eyeCatch>MEMD</eyeCatch>
+        <physicalOffset>0x385B000</physicalOffset>
+        <physicalRegionSize>0x20000</physicalRegionSize>
+        <side>sideless</side>
+        <sha512Version/>
+        <ecc/>
+    </section>
+    <section>
+        <description>Secureboot Test Load (12K)</description>
+        <eyeCatch>TESTLOAD</eyeCatch>
```
Comment (dcrowell77) [inline p9Layouts/axonePnorLayout_64.xml]: no test sections

File: p9Layouts/axonePnorLayout_64.xml [hunk 4]
```diff
@@ -0,0 +1,335 @@
...
+    </section>
+    <section>
+        <description>Guard Data (20K)</description>
+        <eyeCatch>GUARD</eyeCatch>
+        <physicalOffset>0x2C000</physicalOffset>
+        <physicalRegionSize>0x5000</physicalRegionSize>
+        <side>sideless</side>
+        <ecc/>
+    </section>
+    <section>
```
Comment (dcrowell77) [inline p9Layouts/axonePnorLayout_64.xml]: There is a change in flight to remove a couple of these vpd cache sections, might as well incorporate those changes

**Comment by dcrowell77** [comment]
Need to add the extra OP tags that we have in the current P9 layout.
<reprovision/>
<clearOnEccErr/>
<preserved/>

Missing a few partitions too : VERSION, BMC_INV, CAPP, ATTR_TEMP, ATTR_PERM, IMA_CATALOG, UVISOR (?)

File: p9Layouts/axonePnorLayout_64.xml (hunk 4 above)
Comment (mabaiocchi) [reply p9Layouts/axonePnorLayout_64.xml]: I did remove the CVPD, MVPD, and DJVPD sections.

**Comment by mabaiocchi** [comment]
@wghoffa @dcrowell77 @crgeddes @IlyaSmirnov91 Please review these PNOR repo changes needed for axone/swift support.

**Pull request merged**